servant/src/Servant/API/Empty.hs

```diff
@@ -0,0 +1,10 @@
+{-# LANGUAGE DeriveDataTypeable #-}
+{-# OPTIONS_HADDOCK not-home    #-}
+module Servant.API.Empty(EmptyAPI(..)) where
+
+import           Data.Typeable    (Typeable)
+import           Prelude ()
+import           Prelude.Compat
+
+-- | An empty API: one which serves nothing.
```

Comment:
Yes, I'm happy to say that interpretations should treat it as the unit of `:<|>` as much as possible.